x-pack/plugins/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -213,6 +217,7 @@ export const AnomaliesTable = ({
   dateRange = DEFAULT_DATE_RANGE,
   hideDatePicker = false,
   request$,
+  isHostsPage,
```

Comment:
I wouldn't tie this property name to a page. I think is more valuable to have a toggle to turn the select group on/off.

- `showSelectGroup`
- `withSelectGroup`